sandbox/src/src/index.tsx

```diff
@@ -434,7 +442,7 @@ export const Shadow: React.FC<ShadowProps> = ({
           ? (<Path fill={startColorWoOpacity} fillOpacity={startColorOpacity} d={`M0,${topLeft} v${height - bottomLeft - topLeft} h${bottomLeft} v${bottomLeft} h${width - bottomLeft - bottomRight} v${-bottomRight} h${bottomRight} v${-height + bottomRight + topRight} h${-topRight} v${-topRight} h${-width + topLeft + topRight} v${topLeft} Z`}/>)
           : (<>
             <Defs>
-              <Mask id='maskPaintBelow'>
+              <Mask id={`maskPaintBelow.${gradientIdSuffix}`}>
```

Comment:
Not certain if this is necessary, but it keeps the code consistent, won't affect normal function, and makes it easier to identify which shadow the mask belongs to when inspecting elements.